src/vs/workbench/contrib/tasks/browser/abstractTaskService.ts

```diff
@@ -597,6 +604,13 @@ export abstract class AbstractTaskService extends Disposable implements ITaskSer
 		return this._recentlyUsedTasks;
```

Comment:
Instead of modifying map.ts let's just iterate through `values` in reverse order.